### DISCUSSION TITLE
关于空格键暂停的建议

### DISCUSSION THREAD

**Discussion by qw4096** [the1812/Bilibili-Evolved#1325] [功能建议 / Ideas]
现在点击网页的其他地方后，再敲空格，它就变成了翻页，建议调整一下，可以设置为出发焦点在文本框里，否则空格都是暂停播放

**Comment by the1812** [comment]
开启`快捷键扩展`就有这功能

**Reply by Geeyun-JY3** [reply]
将这个也写进设置面板的功能提示吧，像 [功能列表](https://github.com/the1812/Bilibili-Evolved/blob/master/features.md) 里说的一样。
***
> (原生快捷键需要聚焦到播放器才能用, 除了 <kbd>f</kbd> 和 <kbd>space</kbd> 快捷键会被额外再扩展到全网页范围)

话说 [功能列表](https://github.com/the1812/Bilibili-Evolved/blob/master/features.md) 里的这个表述是不是过时了，这俩在无插件状态下都不会被额外再扩展到全网页范围。

**Reply by the1812** [reply]
啊 可能我描述的太迷惑了 开着快捷键扩展功能才会进行"额外扩展"

**Reply by Geeyun-JY3** [reply]
> 开着快捷键扩展功能才会进行"额外扩展"

soga
那 <kbd>f</kbd> 的额外扩展目前已经失效，#1287 也有提到
